Lib/test/test_tarfile.py

```diff
@@ -3056,6 +3057,94 @@ def test_keyword_only(self, mock_geteuid):
                               tarfl.extract, filename_1, TEMPDIR, False, True)
 
 
+_can_chmod_set_sticky = None
+
+
+def can_chmod_set_sticky():
+    """Can chmod set the sticky bit on a file?"""
```

Comment:
Please explain in the docstring that it returns False if chmod() doesn't fail but ignores the sticky bit.